src/_includes/footer.html

```diff
@@ -1,33 +1,32 @@
-<section class="bg-dark navbar d-print-none">
+<section class="{{ page.footer-class }} bg-dark navbar d-print-none">
```

Comment:
I had to add this to get the footer to stay at the bottom on the Customer stories page. The way the `Our impact` section is absolutely positioned breaks everything below it.